aioaws/core.py

```diff
@@ -5,10 +5,10 @@
 from binascii import hexlify
 from datetime import datetime
 from functools import reduce
-from typing import TYPE_CHECKING, Any, Dict, List, Literal, Optional, Tuple
+from typing import TYPE_CHECKING, Any, Dict, Generator, List, Literal, Optional, Tuple
 from urllib.parse import quote as url_quote
 
-from httpx import URL, AsyncClient, Response
+import httpx
```

Comment:
Sorry to be be boring, let's reverse this to minimise the changes.

Reply:
Looks like `Request` is the only other thing you need?

Reply:
addressed